protolite-well-known-types/gradle.properties

```diff
@@ -1,2 +1,3 @@
-version=18.0.1
+# This is done to prevent issues with transitive dependencies
```

Comment:
Create an internal tracking bug to reference here. Then you can make it look more like a TODO (suggested wording)

# IMPORTANT (b/1111111): Keep version and latestReleasedVersion in sync unless you are releasing a new version of the library to prevent issues with transitive dependencies.